src/main/java/bitflow4j/Header.java

```diff
@@ -16,26 +18,19 @@ public Header(String[] header) {
         this.header = header;
     }
 
-    public boolean hasChanged(Header oldHeader) {
```

Comment:
This method should remain imo. It is used a lot throughout all projects that build uppon bitflow4j. Inside, it could call equals.